### DISCUSSION TITLE
Seq authentication with OpenID

### DISCUSSION THREAD

**Discussion by romeosta** [datalust/seq-tickets#1863] [Help]
Hello,

I have deployed Seq on a self-managed k8s cluster with Kubeadm. I and trying to configure OpenID authentication but it is not going to let me use self-signed certificates. Can you please advise how to skip the certificate verification for this particular case?

Thanks!

**Comment by nblumhardt** [comment]
Hi @romeosta, thanks for your message.

If the problem is that Seq doesn't trust the OIDC provider's certificate (i.e. on the backplane) then using an [init script](docs.datalust.co/docs/init-scripts) to [add the certificate to the container's trust store](https://ubuntu.com/server/docs/security-trust-store) should do the trick.

Hope this helps,
Nick
